index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Abdellah Portfolio</title>
+    <link rel="stylesheet" href="stylesheet.css">
+</head>
+<body>
+    <div class="firstSection">
+        <header>
+            <nav class="navbar">
+                <ul class="myNavLinks">
+                    <li><a href="#">Portfolio</a></li>
+                    <li><a href="#">About</a></li>
+                    <li><a href="#">Contact</a></li>
+                </ul>
+            </nav>
+        </header>
+        <main>
+            <p id="firstPara">Hey there.</p><p id="supportingPara">I'm Abdou.</p>
```

Comment:
- [ ] Please the main title is a `<h1>` tag, not a `<p>` Because `<p>` tag is used to structure text in paragraph.